nautobot/core/templates/generic/object_retrieve.html

```diff
@@ -85,11 +85,13 @@ <h1>
             </a>
         </li>
         {% if object.is_contact_associable_model %}
-            <li role="presentation"{% if request.GET.tab == 'contacts' %} class="active"{% endif %}>
-                <a href="{{ object.get_absolute_url }}#contacts" onclick="switch_tab(this.href)" aria-controls="contacts" role="tab" data-toggle="tab">
-                    Contacts
-                </a>
-            </li>
+            {% if perms.extras.view_contactassociation %}
```

Comment:
Possibly combine the two conditionals in a single line, i.e. `{% if object.is_contact_associable_model and perms.extras.view_contactassociation %}` similar to the dynamic group case below?